sdk/src/main/java/org/coralibre/android/sdk/internal/datatypes/IntervalOfCapturedData.java

```diff
@@ -0,0 +1,13 @@
+package org.coralibre.android.sdk.internal.datatypes;
+
+import java.util.List;
+
+public interface IntervalOfCapturedData {
+
+    ENInterval getInterval();
+
+    List<CapturedData> getCapturedData();
+
+    void add(final CapturedData capturedData);
+
+}
```

Comment:
Maybe this interface can be removed? It has (and will always have) only one implementation

Reply:
I think you are right, I deleted it now. If this was a mistake, it shouldn't be too hard to revert.